### DISCUSSION TITLE
Teclados e Mouses Ergonômicos

### DISCUSSION THREAD

**Discussion by Matheus-de-Souza** [frontendbr/forum#996] [Perguntas e Respostas]
Oi pessoal,

Eu vi que tem uma discussão bem legal sobre cadeiras (#292), mas aí eu queria ver com vocês sobre outro aspecto da ergonomia: os mouses e teclados. Alguém aí usa mouse ou teclado ergonômico? Eu vi que existem mouses verticais e teclados divididos, alguém usa algo assim e recomenda o uso?

**Comment by rodfersou** [comment]
fala Matheus,

Faz 3 anos eu investi em um teclado: https://www.trulyergonomic.com/store/truly-ergonomic-mechanical-keyboard-soft-tactile-kailh-cherry-mx-compatible-brown-keyswitches-229-english

A adaptação foi difícil, acho que levei uns 4 meses para me sentir confortável nesse teclado, mas valeu a pena, as dores na mão simplesmente sumiram quando me acostumei.

O problema é que eu comprei direto no site deles, e somando imposto e valor do fedex paguei praticamente 80% a mais do valor do produto... Foi uma facada na época..

**Comment by felipefialho** [comment]
Quem manja muito é o @grippado

**Comment by grippado** [comment]
Opa, valeu a citação @LFeh, sobre o uso de teclados e mouses ergonômicos ou não, daria pra ficar escrevendo aqui por algumas horas, mas a conclusão seria: "gosto pessoal é a melhor escolha sempre".

Maaaaas, existem várias pontos a sertem levantados na escolha de um teclado e um mouse "ideal", vou tentar ser mais resumido, pq adoro o assunto e sou muito prolixo, hahahahah

-----
#### Teclados:
- Posição dos braços:
Voce digita apoiando as mãos ou o punho, ou o antebraço? você gosta de digitar com o braço flexionado e com o teclado bem próximo do seu corpo, ou com o braço esticado e com a distância máxima possível que conseguir!?
Teclados tem 2 coisas primordiais sobre gosto e costume:
- pressão das teclas
o senso comum é: quanto mais leve melhor, certo? mas isso é um erro, pois teclas leves demais fazem com que não se tenha controle de força aplicada, causando as vezes excesso de uso a musculatura da mão e dos dedos, e acelerando absurdamente o tempo qual a maldita LTC (mais conhecida como L.E.R), por isso, existem os teclados mecânicos que tem vários tipos de switches (gatilho de acionamento individual das teclas), que variam entre, peso, pressão, força, curso, som de resposta e sensação de resposta táctil.
Recomendo o Guia da PHE sobre os tipos de switches existentes e suas variações e propriedades pra quem quiser saber mais sobre: https://www.oficialphe.com.br/03/12/2016/como-escolher-switch-teclado-mecanico/
(eu uso um teclado mecânico com Switch Cherry MX Brown, PBT duplo e do tamanho TKL, se vc ler o guia do link acima, vai entender certinho sobre o que estou falando)
- disposição de teclas
teclados separados, com elevação, com diferenças de tamanhos de teclas, etc, podem parecer uma ótima alternativa, mas pense, se vc usa um teclado completamente adaptado a você, e um dia não puder usar ele, vc vai sofrer as graves consequência de simplesmente se perder com posicionamento de mãos e disposição de acentuação/atalhos, que leva a outro ponto;
- layout
esse, pra mim, é também um ponto muito importante, que é se eu seu teclado vai ser QWERTY, DVORAK, COLEMAK, WORKMAN, e variações mil de cada um deles, tem muitas nuances a mais sobre o uso desses layouts, e a diferença passa longe de ser só ter ou não o cedilha (odeio teclados ABNT/ABNT2), tem a ver com posição de teclas mais usadas, forma de uso de sinais em específico, e isso sim, pode fazer muito a diferença, (uma experiencia que fiz em 2015, foi utilizar por 3 meses, um teclado com layout DVORAK, na primeira semana, eu achava que nunca iria me acostumar, na segunda começei a me habituar, na terceira eu já estava confortável, e 1 mês depois eu realmente queria usar aquele teclado pra sempre <3), quando voltei pro QWERTY, senti um pouco de dificuldade com a diferença de uso, principalmente de ter que usar sinais comuns no desenvolvimento JavaScript que eu considero muito melhores no DVORAK, do que no QWERTY.
Layout DVORAK: (US Simplified DVORAK)
![image](https://user-images.githubusercontent.com/550632/37675203-70a489a2-2c53-11e8-9080-e32f283df367.png)
Layout QWERTY: (US - International)
![image](https://user-images.githubusercontent.com/550632/37675189-6277f6f2-2c53-11e8-9799-e7f83db42aed.png)

-----
#### Mouses
pra mouse existem mais vertentes possíveis de possibilidades, mas vou me ater somente a três, que acredito ser as que mais influenciam pra quem usa muito o mouse pa trabalhar:
- peso
"quanto mais pesado, mais fácil de usar", isso pode parecer contraditório, mas já foi comprovado que o mouse mais pesado, diminui sua aceleração, e consequentemente, aumenta sua precisão, e não, o mouse não é melhor ou pior pq tem 1200 ou 24000 dpi, isso só é um fator altamente influenciável se vc é um jogar profissional de qualquer tipo de jogo que vc precisa ficar acertando "head shots" nos adversários, mas também pode variar, pois existem pró-players que jogam com sensibilidade gigantesca, e outro com uma que necessita quase fazer academia, de tanto que precisam movimentar os braços, os que usam sensibilidade mais baixa, normalmente usam mouses mais leves, pra diminuir o esforço, mas a grande maioria, prefere um mouse pesado e com sensibilidade moderada (e claro, uma taxa de DPIs apropriada), para o trabalho, é mais ou menos a mesma coisa, só que com certeza o esforço é muito menor, pois não exige tanta precisão quanto um jogo, eu particularmente, prefiro mouses pesados com sensibilidade maior, pra evitar movimentar pouco mais que alguns centimetros da mão, o outro fator que influencia muito é a;
- pegada
Palm, claw, finger, 

![](http://lolguia.com/wp-content/uploads/2015/05/palmclawfingertipmouse.png)
![](https://www.clubedohardware.com.br/uploads/monthly_2017_06/grip3.png.74d62da8bbdab68f9de1f800f23b56ea.png)

essas são as três mais comuns, e que com certeza 99% das pessoas tem, cada uma delas fala muito sobre a forma como se manuseia o mouse, e tem direta influencia do terceiro ponto dessa lista que é;
- tamanho
o tamanho não tem nenhuma relação com o peso, e vice versa, e ele é o principal influenciador do seu tipo de pegada, você prefere um mouse pequeno e muito leve, ou algum mais comprido e leve?
gosta de um mouse mais alto e que fique mais confortável de encher a mão toda?
isso realmente é algo muito pessoal, não existe "o melhor mouse", ou "a melhor pegada", existe aquela que você consegue usar o dia todo sem que isso te custe uma tendinite ou um estiramento muscular depois de algum tempo, eu gosto de mouse largo e pesado, pra pegada palm, é meu estilo de uso, e você? sabe qual é o seu?!

Espero que seja útil!

#pas

**Comment by Matheus-de-Souza** [comment]
@grippado Muito obrigado pelas dicas! Certamente serão de grande valor para quem está interessado no assunto. Você poderia indicar onde comprou os equipamentos que você usa? Tenho muita curiosidade sobre o teclado dvorak, em minhas pesquisas não consegui encontrar um teclado assim para comprar.

**Comment by Matheus-de-Souza** [comment]
@rodfersou Muito obrigado pela contribuição! Eu estou pensando nesse assunto de ergonomia justamente por essa questão das dores. Tenho sentido nos ombros há um bom tempo, espero que eu consiga encontrar um conjunto mouse/teclado que também me ajude a diminuir isso. Achei muito interessante essa dica de teclado menor, que possibilita usar o mouse mais perto.

**Comment by akfzambrana** [comment]
nossa @grippado muito legal tua resposta, adorei :heart: 

Uma pequena contribuição sobre minha experiencia...

Eu não uso teclado ergonomico, mas uso um teclado mecanico (um redragon sem numpad) e depois de 1 ano de uso posso afirmar q a diminuição da dor nos braços e ombros é brutal. 

Na minha vida eu troquei o teclado pra mecanico e o mouse para uma "mesa digitalizadora" estas duas mudanças foram essenciais para eu ter menos dor no dia a dia. (Inclusive recentemente passei a usar o mouse de novo pq o driver da mesa digitalizadora ta bugado no meu fedora e a dor no meu pulso e ombro direito voltou :sob: )

**Comment by caioincau** [comment]
Depois que acostumei com o trackpad do Mac, não consigo usar mouse mais :( 

Quanto ao teclado, gosto de teclado no estilo chiclete, hoje em dia quando uso teclado/"mouse" externo, uso o Apple Keyboard e o Magic Trackpad.

**Comment by zhengui67** [comment]
Mesa digitalizadora é um equipamento muito mais profissional, a textura de papel é muito realista, tem botões pra trocar ferramenta, dar zoom, etc, a tela do computador é muito maior, até o macbook air é maior... e a sensibilidade de pressão é um diferencial sem comparação. Mas usar um iPad pra desenho está longe de ser "de brincadeira".
melhores opções para começar, na minha opinião:

XP-Pen DECO 03 graphics tablet
 5080 lpi , e 8192 de pressão ,  Área Ativa: 25,4 x 15,9 cm (10” x 5,62”) , Área ativa maior fica mais confortável usar em resoluções maiores. 
https://www.xp-pen.com/goods/show/id/314.html
 eu tenho uma DECO 03 é é bem boa e dá para fazer muita coisa nela no Toon Boom, Zbrush e Ilu. Depois que vc tiver mais estabelecido invista numa XP-Pen Artist  .
